src/nviz/image.py

```diff
@@ -111,7 +111,7 @@ def tiff_to_zarr(  # noqa: PLR0913
     output_path: str,
     channel_map: Dict[str, str],
     scaling_values: Union[List[int], Tuple[int]],
-    ignore: Optional[List[str]],
+    ignore: Optional[List[str]] = ["Merge"],
```

Comment:
It's unclear to me what "Merge" is, maybe it's ok? Could also add some phrasing to the docstring

Reply:
Thanks, I'll add some content to the docstring here. For the data this work was created from we found that we wanted to ignore the "Merged" data which I believe merged multiple channels together. There's a chance this gets removed as a static default in the future if we find the pattern becomes problematic.